example/README.md

```diff
@@ -0,0 +1,36 @@
+# example
+
+The docker-compose environment consists of:
+
+- a standalone Flink cluster (`jobmanager` and `taskmanager` containers).
+- an "edge" node from which Flink jobs can be scheduled (`flink-sql-runner-node` container).
+- `MinIO` instance which serves as AWS S3 substitute.
+- a short-lived container `createbuckets` which configures minio, namely, creates necessary buckets, policies and
+  users.
+- Elastic instance.
+- a short-lived container `elastic-data` which puts test data into Elastic.
+- a Jupyter Notebook container.
+
+Flink UI can be accessed at `http://localhost:8081`.
+Minio UI can be accessed at `http://localhost:9001` (`user`/`password`).
+Jupyter UI can be accessed at `http://localhost:8888`.
+
+## Build
+
+1. Build java artifacts first.
+   ```bash
```

Comment:
For everything to go smoothly, we need to be in flink-elastic-catalog directory, not in example directory.
Add `cd ../` before running `mvn clean install` ...
Besides that LGTM.

Reply:
fixed